app/views/site/_study_title_bar.html.erb

```diff
@@ -1,3 +1,5 @@
+<% content_for(:html_title) { "#{@study.accession} - Single Cell Portal" } %>
```

Comment:
Conditionally adding study names to these titles would improve SEO and human glanceability.

Only showing accession for _private_ studies is reasonable.  However, for public studies, no such privacy concerns exist.  In those cases it'd enhance search engine optimization to also show the study's name.  Names are also more useful than accessions for humans, e.g. tracing one's past in browser history -- or even glancing at currently-open tab titles; the first few characters of a name are usually better hooks for recall than an accession.

So something like "if private, use accession; if public, use name and accession" would enrich this work's value.